swaggerpay/payswaggerlatest.json

```diff
@@ -1072,7 +1072,62 @@
           }
         }
       }
+    },
+
+    "/ServiceFee/GetServiceFee": {
+      "post": {
+        "tags": [
+          "ServiceFee"
+        ],
+        "summary": "Get ServiceFee",
+        "description": "This endpoint is used to get the service fee.\n\n## Request\n\n| Property                  | Description  |\r\n| ------------------------- | ------------------------------------------------------------------------|\r\n|PaymentType               | Payment Type can be any of \"CARD\", \"ACH\", \"EFT\".                      |\r\n| Calling System                 | Calling System can be any of \"CUSTOMER PORTAL\", \"EXTERNAL\"         |\r\n| Currency                     |The currency associated to the payment request. |\r\n|PrincipalAmount                     |The base amount associated to the payment request (This field is optional for Tokenization).\n\n## Response\n\n| Property                  | Description  |\r\n| ------------------------- | ------------------------------------------------------------------------|\r\n| ServiceFee                      | The service fee associated to the response.        |\r\n| Percentage               |  The percentage associated to the response.                              |\r\n| PrincipalAmount                     |The base amount associated to the payment request (This field is optional for Tokenization).        |\r\n| Total                     |  The total associated to the response. |\r\n| TermsAndConditions                     | The terms and conditions associated to the response.",
```

Comment:
can we move these as inline description to request and response schema instead?

Reply:
We are trying to make consistency, can not make a change now